shared/teams/render.js

```diff
@@ -0,0 +1,54 @@
+// @flow
+import * as React from 'react'
+import {ClickableBox, Box, Icon, ScrollView, Text} from '../common-adapters'
+import {globalColors, globalStyles} from '../styles'
+
+import type {Props} from './render'
+
+function TeamsRender(props: Props) {
+  // TODO: Find right icon for "Join a team" button.
+  return (
+    <Box style={{...globalStyles.flexBoxColumn, alignItems: 'center', height: '100%'}}>
+      <Box style={{...globalStyles.flexBoxRow, alignItems: 'center', height: 48}}>
+        <ClickableBox
+          onClick={props.onCreateTeam}
+          style={{...globalStyles.flexBoxRow, alignItems: 'center', marginRight: 40}}
+        >
+          <Icon type="iconfont-new" style={{color: globalColors.blue}} />
+          <Text type="HeaderLink" style={{padding: 5}}>Create a team</Text>
+        </ClickableBox>
+
+        <ClickableBox onClick={props.onJoinTeam} style={{...globalStyles.flexBoxRow, alignItems: 'center'}}>
```

Comment:
Its likely worth it to break these out into sub components (in this file)
so you can do
```
const Banner = ({}) => ...
const List = ({}) => ...
const BetaNote = () => 

and then this list is more like
<Box>
<Banner whateverprops={true}/>
<List />
<BetaNote />
```

Reply:
Done.